docs/csharp/methods.md

```diff
@@ -257,7 +257,7 @@ An async method can't declare any [ref](https://msdn.microsoft.com/library/14akc
 
  For more information about async methods, see [Asynchronous Programming with Async and Await](https://msdn.microsoft.com/library/mt674882.aspx), [Control Flow in Async Programs](https://msdn.microsoft.com/library/mt674892.aspx), and [Async Return Types](https://msdn.microsoft.com/library/mt674893.aspx).
 
-## <a name="expr" /> Expression bodied members ##
+## <a name="expr" /> Expression-bodied members ##
```

Comment:
nit: the anchors need to be before the section title, otherwise it creates anchors like this: #a-nameexpr--expression-bodied-members instead of #expression-bodied-members
Maybe this is still something leftover from the migration? I know I've asked them to fix this for the upcoming migration.